backend/lib/mes-aides/emails/email-render.ts

```diff
@@ -40,17 +41,27 @@ const textTemplates = {
   [EmailType.simulationUsefulness]: simulationUsefulnessTextTemplate,
 }
 
+const buildAPIEmailRenderPath = (followupId, emailType) => {
+  const payload = {
+    followupId,
+    emailType,
+  }
+  return `/api/email/${jwt.sign(payload, config.sessionSecret)}`
```

Comment:
J'avoue je trouve ça compliqué avec le JWT.

Pourquoi ne pas avoir fait le dev sur 
```suggestion
  return `/api/simulations/:simulationId/followup/:followupId`
```

en sécurisant avec la logique existante (ie cookie) ?

Reply:
En fait l'objectif qu'on souhaitait atteindre c'est : 
- Pouvoir identifier le followup de l'email
- Pouvoir s'assurer qu'on a bien déjà envoyé l'email pour l'afficher
(post initial [ici](https://github.com/betagouv/aides-jeunes/pull/3680#discussion_r1190822052))

Chose qu'on arrive à faire avec le JWT sans rajouter un token par email dans le followup. (car on met l'email type dans la payload avant de signer). 

Je ne vois pas trop comment faire avec la logique cookie (sachant qu'on a pas de moyen de s'assurer qu'on est sur le même navigateur que là où la simulation a été faite) mais on peut en parler.

Reply:
> Pourquoi ne pas avoir fait le dev sur
>   return `/api/simulations/:simulationId/followup/:followupId`

J'ai passé un peu de temps à essayer de comprendre, mais j'arrive toujours pas à saisir pourquoi on aurait besoin de l'identifiant de la simulation dans la route pour identifier l'email alors que l'id du followup suffit et que l'id de la simulation est inclu dans ce dernier? Comment on identifie le type de l'email sans le mettre dans un query param ?